src/fft.js

```diff
@@ -99,7 +110,6 @@ define(function (require) {
     // default connections to p5sound fftMeter
     p5sound.fftMeter.connect(this.analyser);
 
-    this.analyser.smoothingTimeConstant = this.smoothing;
```

Comment:
I think we still need to set this, and maybe we can do the `Object.defineProperty` below this line?